app/controllers/events/view/settings.js

```diff
@@ -0,0 +1,47 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+
+  actions: {
+    openEventTransferModal(id, name) {
+      this.setProperties({
+        'isEventTransferModalOpen' : true,
+        'confirmEventName'         : '',
+        'eventId'                  : id,
+        'eventName'                : name
+      });
+    },
+    openConfirmEventTransferModal() {
+      this.setProperties({
+        'isEventTransferModalOpen'        : false,
+        'isConfirmEventTransferModalOpen' : true,
+        'checked'                         : false
+      });
+    },
+    transferEvent(event, data) {
+      this.set('isLoading', true);
+      let payload = {
+        'data': {
+          event,
+          'new-organizer': data.newOrganizerEmail
+        }
+      };
+      this.loader
```

Comment:
`return this.loader`
Also, try and work out the agreed upon implementation!

Reply:
@uds5501 did your backend code is ready? Do your backend code require `return this.loader` and is it working?

Reply:
@uds5501 I tried this 
```js
return this.loader
        .patch('/transfer-ownership', JSON.stringify(payload))
```
but params in my network tab is empty. Can you please check?

Reply:
@shreyanshdwivedi there must be an error. what does it say?

Reply:
@uds5501 the frontend is now sending data to server :smile: 
```
data: {
    event : {…}
    new-organizer : s@gmail.com
}
```
The data contains event info and new organizer email.
![Screenshot from 2019-06-04 18-03-41](https://user-images.githubusercontent.com/25428397/58879397-3839d580-86f3-11e9-9235-2bab1b43d112.png)

Currenlty error being raised -
`The method is not allowed for the requested URL.`

Reply:
@shreyanshdwivedi we need to send this payload to `events/<event_id>` Please see if it is working in that case